qiskit/pulse/schedule.py

```diff
@@ -80,87 +81,24 @@ def __init__(self,
                 name += '-{}'.format(mp.current_process().pid)
```

Comment:
I didn't implement this, but I guess multiprocessing in scheduler (perhaps this is not implemented now) may generate duplicated name since schedule is initialized with different threads?

Reply:
Seems extremely fragile 😬